docs/hacking/compiling.rst

```diff
@@ -206,6 +207,12 @@ To build Angband with the GCU front end::
     cmake -DSUPPORT_GCU_FRONTEND=ON ..
     make
 
+To build Angband with the Windows front end::
```

Comment:
As above, this is for Linux/other Unix so need cross-compiling for Windows.  So I'd just refer to the cross-compiling for Windows part here.

Reply:
Reverted